sor/src/main/java/com/bazaarvoice/emodb/sor/db/astyanax/AstyanaxDataWriterDAO.java

```diff
@@ -174,6 +184,32 @@ private void writeAll(Map<BatchKey, List<BatchUpdate>> batchMap, UpdateListener
         }
     }
 
+    private void putDeltaColumn(ColumnListMutation mutation, UUID changeId, ByteBuffer encodedDelta, int deltaSize) {
+        int numBlocks = (deltaSize + _deltaBlockSize - 1) / _deltaBlockSize;
```

Comment:
I guess I can do that. That being said, I'm not sure it has any utility, as we already have a check in write to ensure the delta isnt large than MAX_DELTA_SIZE